src/main/java/service/activities/BalanceTransactionHandler.java

```diff
@@ -0,0 +1,18 @@
+package service.activities;
+
+import db.FruitShopStorage;
+import model.FruitTransaction;
+
+public class BalanceTransactionHandler implements TransactionHandler {
+
+    @Override
+    public void executeTransaction(FruitTransaction fruitTransaction) {
+        String fruit = fruitTransaction.getFruit();
+        int quantity = fruitTransaction.getQuantity();
+        if (quantity < 0) {
+            throw new RuntimeException("Initial quantity cannot be negative for fruit: " + fruit);
+        }
```

Comment:
We have this check in all handlers. This is code duplication. How can we fix it? 
Ask in chat in case of no ideas)